adcc/LazyMp.py

```diff
@@ -179,3 +277,22 @@ def mp2_dipole_moment(self):
         mp2corr = -np.array([product_trace(comp, self.mp2_diffdm)
                              for comp in dipole_integrals])
         return refstate.dipole_moment + mp2corr
+
+
+#
+# Register cvs_p0 intermedites
+#
+@register_as_intermediate
+def cvs_p0_oo(hf, mp, intermediates):
```

Comment:
Make connection to mp2_diffdm expressions clear by mutual comments (easier to refactor later because otherwise you might miss an instance).